lib/qxcli/commands/Compile.js

```diff
@@ -472,87 +275,96 @@ qx.Class.define("qxcli.commands.Compile", {
           if (src) {
             src.forEach(function(elem) {
-              if (!qx.lang.Array.contains(dest, src))
+              if (!qx.lang.Array.contains(dest, src)) {
                 dest.push(elem);
+              }
             });
           }
         });
         return dest;
       }
 
       var addCreatedAt = data.target["addCreatedAt"] || t.argv["addCreatedAt"];
-      if (addCreatedAt)
+      if (addCreatedAt) {
         maker.getAnalyser().setAddCreatedAt(true);
-
+      }
       var appNames = null;
-      if (t.argv["app-name"])
-        appNames = t.argv["app-name"].split(',');
-      
+      if (t.argv["app-name"]) {
+        appNames = t.argv["app-name"].split(",");
+      }
       data.applications.forEach(function(appData, appIndex) {
-        if (appNames && appNames.indexOf(appData.name) == -1)
+        if (appNames && appNames.indexOf(appData.name) == -1) {
           return;
+        }
         var app = new qxcompiler.app.Application(appData["class"]);
-        [ "type", "theme", "name", "environment", "outputPath", "loaderTemplate"]
-          .forEach((name) => {
+        ["type", "theme", "name", "environment", "outputPath", "loaderTemplate"]
+          .forEach(name => {
             if (appData[name] !== undefined) {
               var fname = "set" + qx.lang.String.firstUp(name);
               app[fname](appData[name]);
             }
           });
-        if (appData.uri)
+        if (appData.uri) {
           app.setSourceUri(appData.uri);
-        if (appData.title)
+        }
+        if (appData.title) {
           app.setTitle(appData.title);
-        
+        }
         var minify = appData["minify"] || data.target["minify"] || t.argv["minify"];
-        if (typeof minify == "boolean")
+        if (typeof minify == "boolean") {
           minify = minify ? "minify" : "off";
-        if (!minify)
+        }
+        if (!minify) {
           minify = "mangle";
-        if (typeof target.setMinify == "function")
+        }
+        if (typeof target.setMinify == "function") {
           target.setMinify(minify);
-
+        }
         var parts = appData.parts || data.target.parts || data.parts;
         if (parts) {
           if (!parts.boot)
             throw new qxcli.Utils.UserError("Cannot determine a boot part for application " + (appIndex + 1) + " " + (appData.name||""));
           for (var partName in parts) {
             var partData = parts[partName];
-            var include = typeof partData.include == "String" ? [ partData.include ] : partData.include;
-            var exclude = typeof partData.exclude == "String" ? [ partData.exclude ] : partData.exclude;
+            var include = typeof partData.include == "String"? [partData.include] : partData.include;
+            var exclude = typeof partData.exclude == "String"? [partData.exclude] : partData.exclude;
             var part = new qxcompiler.app.Part(partName, include, exclude).set({
-              combine: !!partData.combine,
```

Comment:
why is this code changing from `!!expr` to `Boolean(expr)`?

Reply:
Run the lint process..